src/client/Microsoft.Identity.Client/Instance/Discovery/RegionAndMtlsDiscoveryProvider.cs

```diff
@@ -10,26 +10,36 @@
 
 namespace Microsoft.Identity.Client.Region
 {
-    internal class RegionDiscoveryProvider : IRegionDiscoveryProvider
+    internal class RegionAndMtlsDiscoveryProvider : IRegionDiscoveryProvider
```

Comment:
Why change the name of this class? This still only does the region discovery.

Reply:
when we started off it was providing a generic host for MTLS as well, but now we only use this as regional. @bgavrilMS ?

Reply:
It handles regional and mtls all in one. I think this is ok, decoupling them would complicate the logic quite a bit.